nbodykit/measurestats.py

```diff
@@ -220,26 +223,26 @@ def bianchi_transfer(data, x, i, j, k=None, offset=[0., 0., 0.]):
         
         # do the FKP painting
         stats = catalog.paint(pm)
-    
+
     # save the fkp density for later
     density = pm.real.copy()
     if rank == 0: logger.info('%s painting done' %paintbrush)
     
     # compute the monopole, A0(k), and save
     pm.r2c()
     transfer(pm, pm.complex)
-    A0 = pm.complex.copy()
     if rank == 0: logger.info('ell = 0 done; 1 r2c completed')
     
+    # save volume for normalization purposes
     volume = pm.BoxSize.prod()
     
     # store the A0, A2, A4 arrays
     result = []
-    result.append(A0*volume)
+    result.append(pm.complex*volume) # this will make a copy
```

Comment:
I would think 

A0 = pm.complex * volume

then result.append(A0) is more intuitive. And uses the same amount of memory.